frontend/src/styles/global.scss

```diff
@@ -15,7 +15,10 @@ Only 400 (`normal`), 500 (`var(--font-medium)`), 600 (`var(--font-semibold)`), o
 @import 'functions';
 @import 'mixins';
 @import 'utility-legacy';
-@import 'utilities';
+@import 'colors';
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
```

Comment:
FF is using some custom flex values. (This UI could probably be simplified rathe than adding the custom stuff)

<img width="1265" alt="Screenshot 2023-12-06 at 09 34 29" src="https://github.com/PostHog/posthog/assets/2536520/955f67e4-a070-4330-be5f-8d500c2ed591">

Reply:
Same on editing a FF - `max-w-1/2` doesn't seem to exist

Reply:
Same applies to new survey - I think its just the missing max width values

Reply:
And again for new early access feature 

Probably fixable with one off values https://tailwindcss.com/docs/min-width#arbitrary-values